src/store.js

```diff
@@ -0,0 +1,15 @@
+import {
+  createStore,
+  applyMiddleware,
+  compose
+} from 'redux';
+import reducer from './reducers';
+import { promiseMiddleware } from 'promise-middleware-redux';
+
+export default createStore(
+  reducer,
+  compose(
+    applyMiddleware(promiseMiddleware),
+    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
compose only take a function, move window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__() down one line and outside of the compose block and you should get your code back up and running!